test/jdk/java/lang/Process/ProcessCloseTest.java

```diff
@@ -0,0 +1,662 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package jdk.java.lang.Process;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.FileDescriptor;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.FilterInputStream;
+import java.io.FilterOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.io.Reader;
+import java.io.UncheckedIOException;
+import java.io.Writer;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+/*
+ * @test
+ * @bug 8336479
+ * @modules java.base/java.io:+open
+ * @summary Tests for Process.close
+ * @run junit/othervm jdk.java.lang.Process.ProcessCloseTest
+ */
+
+public class ProcessCloseTest {
+
+    private final static boolean OS_WINDOWS = System.getProperty("os.name").startsWith("Windows");
+    private static List<String> JAVA_ARGS;
+
+    private static List<String> setupJavaEXE() {
+        String JAVA_HOME = System.getProperty("test.jdk");
+        if (JAVA_HOME == null)
+            JAVA_HOME = System.getProperty("JAVA_HOME");
+        String classPath = System.getProperty("test.class.path");
+        return  List.of(JAVA_HOME + "/bin/java", "-cp", classPath, ProcessCloseTest.class.getName());
```

Comment:
The method name says `setupJavaEXE()`. Was this line here expected to conditionally use `.exe` suffix for Windows, like some other tests do?

Reply:
The .exe extension is not necessary or expected. The method name EXE is short for execution.